### ISSUE TITLE
Make sure all IG have a GitHub repo

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/infrastructure#80]
Below are the interest groups for which the conveners presented conclusions at the closing session of the TDWG 2016 conference. All of these need a space on GitHub to collaborate. I'm tracking the status of this task in this issue.

* [x] Annotations: https://github.com/tdwg/annotations
* [ ] Attribution: (repo still private)
* [ ] Citizen science: no charter yet
* [x] Curriculum: https://github.com/tdwg/curriculum
* [x] Darwin Core: https://github.com/tdwg/dwc `task group`
* [x] Data quality: https://github.com/tdwg/bdq
* [ ] Interactions: no charter yet
* [x] Invasive species: https://github.com/tdwg/gisin should become a `task group`
* [ ] Natural Collection Descriptions: 
* [x] Paleobiology: https://github.com/tdwg/paleo
* [x] RDF task group: https://github.com/tdwg/rdf `task group`
* [ ] Species information: move PlinianCore under TDWG
* [ ] Services: pending
* [ ] TAG: pending
* [x] Vocab maintenance: https://github.com/vocab `task group`

**Comment by peterdesmet** [comment]
@jmacklin, is this "Services" already an IG (with a charter) and is there need for a repo?

**Comment by peterdesmet** [comment]
@jmacklin, seems there is: http://www.tdwg.org/activities/biodiversity-services-clients/charter/. So, is there a need for a repo?

**Comment by peterdesmet** [comment]
Hi @debpaul, you presented Software Carpentry at closing session. Is this an interest group and would you need a repo under TDWG?

**Comment by peterdesmet** [comment]
@ghwhitbread does TAG need a repo or does the current mailing list suffice?

**Comment by debpaul** [comment]
Hi Peter,
No, we're not an IG - we were asked to share feedback b/c local hosts and TDWG were prominent in asking for this workshop - and making it possible :-)
Deb

Sent from shoe (iPhone)

**Comment by peterdesmet** [comment]
@debpaul great, I'll remove it from the above list then. 🇨🇷

**Comment by ghwhitbread** [comment]
@peterdesmet Yes please Peter, TAG will need at least one repos for business and perhaps a second for public access.  TAG mail list seems to have more members than TDWG.

**Comment by peterdesmet** [comment]
@ghwhitbread bread, so at least one private repo?

**Comment by ghwhitbread** [comment]
@peterdesmet Yes. Private repo for TAG.

**Comment by peterdesmet** [comment]
@ghwhitbread done! You should have received an invitation through email.

**Comment by PacoPando** [comment]
Hi @peterdesmet ,

Regarding the Species information: IG, and after ite IG meeting at TDWG2016, this is what is needed:

A repository for the IG, that it is titled as "TSI" (TDWG Specied Information IG http://www.tdwg.org/activities/species-information/charter/)

A repository for Plinian Core, that now becomes a TG under the TSI IG; the two current repositores (sources & documentation) to be merged in one).

The old "Invasive Species Interest Group (ISIG)", becomes a TG under the TSI, and a repository for this is needed.  I assume that Annie Simpson (convener) has talked to you about it

I guess you have to invite me to those repositories to start populating them...

As for reporting, following TAG makes sense to me.   

Thanks

**Comment by peterdesmet** [comment]
Hi @PacoPando, I've created a separate issue for moving the PlinianCore to TDWG: #82 Thus, for this issue remains:

* Creating TSI IG repo. Was there a consensus on using `TSI`? `species-information` might be more straightforward...
* Maybe create an ISIG repo: not 100% sure this is needed. Most (all?) work for this group will be on GISIN, which has its repo at: https://github.com/tdwg/gisin @as6699, what do you think?
